examples/embeddable_examples/public/react_embeddables/saved_book/saved_book_editor.tsx

```diff
@@ -61,18 +61,15 @@ export const openSavedBookEditor = ({
           attributesManager={attributesManager}
           onCancel={() => {
             // set the state back to the initial state and reject
-            attributesManager.authorName.next(initialState.authorName);
-            attributesManager.bookSynopsis.next(initialState.bookSynopsis);
-            attributesManager.bookTitle.next(initialState.bookTitle);
-            attributesManager.numberOfPages.next(initialState.numberOfPages);
+            attributesManager.api.setAuthorName(initialState.authorName);
```

Comment:
You should just be able to call `attributesManager.reinitializeState(initialState)` instead of calling setters individually.